Library/Formula/fig.rb

```diff
@@ -79,6 +79,10 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    
+    # change shebang from system python to env python
+    inreplace libexec/"bin/docker-compose", "#!/usr/bin/python", "#!/usr/bin/env python"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!